src/config-hardware/config-hardware.md

```diff
@@ -0,0 +1,9 @@
+# Connecting & Configuring Hardware Devices
+
+Configure peripherals and hardware devices in the `Settings` application.
+
+- [Configure audio input and output devices](audio-mic.md) - Adjust volume, select speakers and microphones, install third-party applications for advanced  features, and use Terminal commands to display diagnostic information or restart daemons that control audio streams.
+- [Adjust mouse and touchpad settings](mouse-touchpad.md) - Choose a primary mouse button, adjust scroll wheel behavior and speed, and enable or disable mouse features like mouse acceleration and disabling the touch pad while typing.
```

Comment:
`touchpad` is one word in the link, but `touch pad` is two words in the description.